### PR TITLE
refactor: use app instead of vue in index file

### PR DESCRIPTION
**Pull request by luqven**
# Description
TTSIA

### CODE REVIEW

File: src/plugins/vue-imgix/index.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
-import _Vue from 'vue';
+import _App from './App.vue';
```
Comment (frederickfogerty) [inline src/plugins/vue-imgix/index.ts:1]: What does `./App.vue` resolve to?

**Review by frederickfogerty — APPROVED** [review]
One comment but otherwise looks good

File: src/plugins/vue-imgix/index.ts (hunk 1 above)
Comment (luqven) [reply src/plugins/vue-imgix/index.ts:1]: Oh wow, 😬  I forgot to push up a branch that precedes this. I'm going to have to force push here to make sure this has all the app. context. 

Basically, it comes down to `vue-class-component`'s `Vue` export.

```vue
// App.vue
<script>
import { Options, Vue } from "vue-class-component";
import HelloWorld from "./components/HelloWorld.vue";

@Options({
  components: {
    HelloWorld,
  },
})
export default class App extends Vue {}
</script>
```

```typescript
// index.ts
import { createApp } from "vue";
import App from "./App.vue";

createApp(App).mount("#app");

```

Comment (luqven) [reply src/plugins/vue-imgix/index.ts:1]: Some more info on this here https://github.com/vuejs/vue-class-component/issues/406

Comment (frederickfogerty) [reply src/plugins/vue-imgix/index.ts:1]: Thanks for the extra info. Looks like maybe the path for this is wrong?

Comment (luqven) [reply src/plugins/vue-imgix/index.ts:1]: > Thanks for the extra info. Looks like maybe the path for this is wrong?

...💯  🤦🏼

**Pull request closed without merging**